doc/source/cluster/kubernetes/examples/gpu-training-example.md

```diff
@@ -0,0 +1,148 @@
+(kuberay-gpu-training-example)=
+
+# ML training with GPU on Kubernetes
+In this guide, we show you how to run a sample Ray machine learning training workload with GPU on Kubernetes infrastructure. We will run Ray's {ref}`PyTorch image training benchmark <pytorch_gpu_training_benchmark>` with a 1 gigabyte training set. The following script consists of:
+- Step1: Setup a Kubernetes on GCP
+- Step2: Deploy a Ray cluster on Kubernetes with KubeRay (operator)
+- Step3: Run PyTorch image training benchmark 
+
+```shell
+# Step1: Setup Kubernetes cluster on GCP
+# e2-standard-8 => 8 vCPU; 32 GB RAM
```

Comment:
Could you indicate in a comment that this is for a CPU-only head node?

Reply:
Resolved https://github.com/ray-project/ray/pull/28353/commits/18486fcc0677a5fc80593b34c27f44b853b8ee66.